app/models/service_retire_task.rb

```diff
@@ -34,6 +34,7 @@ def create_retire_subtasks(parent_service)
     parent_service.service_resources.collect do |svc_rsc|
       next unless svc_rsc.resource.present? && svc_rsc.resource.respond_to?(:retire_now)
       next if svc_rsc.resource.type.blank?
+      next if svc_rsc.resource.type.demodulize.to_s == "Job"
```

Comment:
Could this logic be moved into the Job subclass and then just ask the question here?  It seem the same could be done for the more complex `ServiceTemplate` check on the next line as well.

Then the check here would start to be more like: `next if svc_rsc.resource.create_retire_subtasks?`

cc @tinaafitz